pom.xml

```diff
@@ -87,6 +87,7 @@
         <really.executable.jar.version>1.5.0</really.executable.jar.version>
         <generext.version>1.0.2</generext.version>
         <hamcrest.version>1.3</hamcrest.version>
+        <mockito.version>2.23.0</mockito.version>
```

Comment:
We should use the version defined in the common pom instead of doing this. @xvrl added this to the common pom recently. Supporting newer Java versions typically requires bumping these and it's painful if we have to do it for every project.

Reply:
@xvrl has already done it: https://github.com/confluentinc/ksql/pull/2145 :)

Reply:
Perfect!